packages/platform/platform-http/src/common/utils/createInjector.ts

```
@@ -13,46 +14,42 @@
+    .addProvider(PlatformAdapter, {
+      useClass: adapterToken
+    })
+    .alias(PlatformAdapter, "PlatformAdapter");
 
-  const instance = inj.get<PlatformAdapter>(PlatformAdapter)!;
+  const adapter = inj.invoke(PlatformAdapter);
+
+  inj.settings.set("PLATFORM_NAME", adapter.NAME);
 
-  instance.providers = [...DEFAULT_PROVIDERS, ...instance.providers];
+  const instance = inj.get<PlatformAdapter>(PlatformAdapter)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)